storagemarket/deal_execution.go

```diff
@@ -47,14 +49,21 @@ func (p *Provider) doDeal(deal *types.ProviderDealState, dh *dealHandler) {
 		return
 	}
 
-	p.dealLogger.Infow(deal.DealUuid, "deal execution initiated", "deal state", *deal)
+	if isRestart {
+		p.dealLogger.Infow(deal.DealUuid, "resuming deal on boost restart")
+	}
```

Comment:
Instead of adding the `isRestart` parameter to this method, can we just add this log line in the calling method if it's a restart?